### PR TITLE
[WIP] Homepage

### PR DESCRIPTION
**Pull request by danielfdsilva**

### CODE REVIEW

File: app/assets/scripts/actions/index.js [hunk 1]
```diff
@@ -26,6 +28,35 @@ export function createFieldReport (payload) {
...
+  const f = buildAPIQS(filters);
+
+  return fetchJSON(`/api/v1/surge_alert/?${f}`, GET_SURGE_ALERTS, withToken());
+}
+
+export const GET_APPEALS_LIST = 'GET_APPEALS_LIST';
+export function getAppealsList () {
+  const f = buildAPIQS({
+    end_date__gt: DateTime.local().toISODate(),
+    limit: 100
```
Comment (dereklieu) [inline app/assets/scripts/actions/index.js]: What's your plan for this, and the recursive function below, with real data? I feel like setting the limit to `0` should be ok now that we're using reasonable development data?

File: app/assets/scripts/components/connected/alerts-table.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    }
+  }
+
+  requestResults () {
+    this.props._getSurgeAlerts(this.state.page);
+  }
+
+  handlePageChange (page) {
+    this.setState({ page: page.selected + 1 }, () => {
+      showGlobalLoading();
```
Comment (dereklieu) [inline app/assets/scripts/components/connected/alerts-table.js]: Could just put `showGlobalLoading` in the `requestResults` function.

File: app/assets/scripts/utils/country-centroids.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+  WS: [ -172.11331132714278, -13.786946593957028 ],
+  YE: [ 47.1086302234567, 14.56738963876404 ],
+  ZA: [ 24.703232827543786, -29.26925840866997 ],
+  ZM: [ 28.489715084919187, -13.149159999209038 ],
+  ZW: [ 29.389317242443635, -18.5589791387905 ]
+};
+
+export function getCentroid (iso) {
+  // Centroid or null island.
+  return centroids[iso.toUpperCase()] || [0, 0];
```
Comment (dereklieu) [inline app/assets/scripts/utils/country-centroids.js]: We might want to have something here that will prevent mapping things to `[0, 0]`. Maybe returning false would make it easier to catch those?

**Comment by dereklieu** [comment]
Overall this is coming along really well. The map data isn't showing up for me though, is it supposed to?

File: app/assets/scripts/actions/index.js (hunk 1 above)
Comment (danielfdsilva) [reply app/assets/scripts/actions/index.js]: It's just a matter of removing the recursion block.

File: app/assets/scripts/utils/country-centroids.js (hunk 3 above)
Comment (danielfdsilva) [reply app/assets/scripts/utils/country-centroids.js]: What should be the desired behavior? If we have no centroid don't show the data?
I used null island to present the app from breaking.

**Comment by danielfdsilva** [comment]
@dereklieu map should be showing data.
image

**Pull request merged**